### PR TITLE
Helpers functions modification

### PR DESCRIPTION
**Pull request by ebenahar**
- Changed create_pvc() to instanciate PVC
  - Modifications in create_pod()

### CODE REVIEW

**Comment by ebenahar** [comment]
tests/manage/test_pvc_deletion_during_io.py::TestDeletePVCWhileRunningIO::test_run_io_and_delete_pvc 
------------------------------------------------------------------------------------------------- live log setup --------------------------------------------------------------------------------------------------
20:53:48 - MainThread - root - INFO - RUN ID: 1562003603
20:53:48 - MainThread - root - INFO - All logs are located under /tmp/logs
20:53:48 - MainThread - tests.conftest - INFO - Running OCS basic installation
20:53:48 - MainThread - ocs_ci.ocs.openshift_ops - INFO - Testing access to cluster with /home/ebenahar/ebenahar-ocs/auth/kubeconfig
20:53:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc cluster-info
20:53:49 - MainThread - ocs_ci.ocs.openshift_ops - INFO - Access to cluster is OK!
20:53:49 - MainThread - tests.conftest - INFO - The installation is skipped because the cluster is running
20:53:49 - Dummy-1 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod -A -o yaml
20:53:49 - Dummy-2 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get StorageClass -A -o yaml
20:53:49 - Dummy-3 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get CephFileSystem -A -o yaml
20:53:49 - Dummy-4 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get CephBlockPool -A -o yaml
20:53:49 - Dummy-5 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolume -A -o yaml
20:53:49 - Dummy-6 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolumeClaim -A -o yaml
20:53:49 - Dummy-7 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Secret -A -o yaml
20:53:49 - Dummy-8 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Namespace -A -o yaml
20:54:33 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
20:54:35 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-s6xtv ceph auth get-key client.admin --format json-pretty
20:54:42 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Secret with name secret-test-0120544275
20:54:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig create -f /tmp/Secret8k_npdm7 -o yaml
20:54:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Secret secret-test-0120544275 -o yaml
20:54:48 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding CephBlockPool with name cbp-test-0120544839
20:54:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig create -f /tmp/CephBlockPoolrc3an2s5 -o yaml
20:54:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get CephBlockPool cbp-test-0120544839 -o yaml
20:54:52 - MainThread - tests.helpers - INFO - Verifying that block pool cbp-test-0120544839 exists
20:54:52 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
20:54:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-s6xtv ceph osd lspools --format json-pretty
20:55:00 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding StorageClass with name storageclass-test-0120550094
20:55:00 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig create -f /tmp/StorageClassk8pe5lr5 -o yaml
20:55:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get StorageClass storageclass-test-0120550094 -o yaml
20:55:04 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name pvc-test-0120550458
20:55:04 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig create -f /tmp/PersistentVolumeClaimu9bjhmu1 -o yaml
20:55:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolumeClaim pvc-test-0120550458 -o yaml
20:55:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolumeClaim pvc-test-0120550458 -o yaml
20:55:10 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Pod with name pod-test-0120551053
20:55:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig create -f /tmp/POD_riy6_iib -o yaml
20:55:13 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod pod-test-0120551053 -o yaml
20:55:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod pod-test-0120551053 -o yaml
20:55:23 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod pod-test-0120551053 -o yaml
20:55:29 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod pod-test-0120551053 -o yaml
20:55:34 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod pod-test-0120551053 -o yaml
-------------------------------------------------------------------------------------------------- live log call --------------------------------------------------------------------------------------------------
20:55:40 - MainThread - ocs_ci.ocs.resources.pod - INFO - Running I/O on pod pod-test-0120551053
20:55:40 - Thread-9 - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig rsh pod-test-0120551053 bash -c "let i=0; while true; do echo Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. >> /var/lib/www/html/test$i; let i++; sleep 0.01; done"
20:55:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig rsh pod-test-0120551053 bash -c "find /var/lib/www/html/test1"
20:55:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig delete PersistentVolumeClaim pvc-test-0120550458 --wait=false
20:55:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolumeClaim pvc-test-0120550458
20:56:04 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig delete Pod pod-test-0120551053
20:56:06 - Thread-9 - ocs_ci.utility.utils - ERROR - CMD error:: command terminated with exit code 137
20:56:06 - Thread-9 - ocs_ci.ocs.resources.pod - INFO - I/O command got terminated as expected
20:56:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolumeClaim pvc-test-0120550458
20:56:14 - MainThread - ocs_ci.utility.utils - ERROR - CMD error:: Error from server (NotFound): persistentvolumeclaims "pvc-test-0120550458" not found
PASSED
------------------------------------------------------------------------------------------------ live log teardown ------------------------------------------------------------------------------------------------
20:56:14 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get StorageClass storageclass-test-0120550094 -o yaml
20:56:15 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig delete StorageClass storageclass-test-0120550094
20:56:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get CephBlockPool cbp-test-0120544839 -o yaml
20:56:19 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig delete CephBlockPool cbp-test-0120544839
20:56:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Secret secret-test-0120544275 -o yaml
20:56:22 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig delete Secret secret-test-0120544275
20:56:24 - Dummy-1 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Pod -A -o yaml
20:56:24 - Dummy-2 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get StorageClass -A -o yaml
20:56:24 - Dummy-5 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get CephFileSystem -A -o yaml
20:56:24 - Dummy-10 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get CephBlockPool -A -o yaml
20:56:24 - Dummy-6 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolume -A -o yaml
20:56:24 - Dummy-8 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get PersistentVolumeClaim -A -o yaml
20:56:24 - Dummy-11 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Secret -A -o yaml
20:56:24 - Dummy-12 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs/auth/kubeconfig get Namespace -A -o yaml


=========================================================================================== 1 passed in 196.73 seconds ============================================================================================

File: tests/fixtures.py [hunk 1]
```diff
@@ -89,10 +88,6 @@ def create_pod(request):
     """
     class_instance = request.node.cls
 
-    pod_data = templating.load_yaml_to_dict(constants.CSI_RBD_POD_YAML)
-    pod_data['metadata']['name'] = helpers.create_unique_resource_name(
-        'test', 'pod'
+    class_instance.pod_obj = helpers.create_pod(
+        interface_type=constants.CEPHBLOCKPOOL, pvc=class_instance.pvc_obj.name
```
Comment (vavuthu) [inline tests/fixtures.py]: since it always creates pod with cephblockpool, can we change fixture name to something like create_pod_cephblockpool

Comment (ebenahar) [reply tests/fixtures.py]: done

**Comment by shreekarSS** [comment]
@ebenahar  LGTM , I tested with these changes. test log: http://pastebin.test.redhat.com/776548

**Pull request closed without merging**